### PR TITLE
#116 notifications lambda

### LINKED ISSUE #116: Notification lambda terraform
## Description
Create a terraform script that creates a lambda function that runs the notification script

## Outcome
A working terraform script that reliably creates and updates the notification lambda on AWS

## User Story
As a developer, I want to be able to quickly create and update the notification lambda function when running the terraform script, so that the AWS resource is easy to maintain and update.

### PR DESCRIPTION
**Pull request by Jakub-Poskrop**
Closes #116 - Created a terraform script for the lambda function that runs the notifications script. Need to terraform apply it as I am encountering issues.

### CODE REVIEW

**Comment by S1mpySloth** [comment]
Copied the code locally, did `terraform plan` and `apply`, lambda was successfully made on my end. 
I used the `-target` flag to just create the lambda, with it's dependencies being automatically included by -target.

File: terraform/dashboard_ecr.tf [hunk 1]
```diff
@@ -6,26 +6,3 @@ resource "aws_ecr_repository" "c14_earthquake_monitor_dashboard_ecr" {
     scan_on_push = true
   }
 }
-
-resource "aws_ecr_lifecycle_policy" "c14_earthquake_monitor_dashboard_ecr_lifecycle_policy" {
```
Comment (S1mpySloth) [inline terraform/dashboard_ecr.tf:10]: Thanks for removing these!

File: terraform/notification_lambda.tf [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            {
+        "Effect": "Allow",
+        "Action": "sns:Publish",
+        "Resource": "arn:aws:sns:eu-west-2:${var.ACCOUNT_ID}:*"
+      }
+    ]
+  })
+}
+
+resource "aws_lambda_function" "c14_earthquake_notification_lambda" {
```
Comment (S1mpySloth) [inline terraform/notification_lambda.tf:47]: All applied correctly, look's good :)

**Review by S1mpySloth — APPROVED** [review]
Changes made as requested, this all applies fine on my machine.

**Review by jiuliangut — APPROVED** [review]
Nice troubleshooting to get this to run locally!

**Review by jiuliangut — APPROVED** [review]

**Review by S1mpySloth — APPROVED** [review]
All resources are being created and lambda works.

**Pull request merged**